miscellaneous-commands/check-duration.md

```diff
@@ -1,31 +1,40 @@
 # check-duration
 
-<TODO: describe>
+This command is equivalent to the previous version’s `/status` command. It allows you to see how long you can set a temporary action. If you [vote for MoonlightBot](/support/upvote-moonlightbot.md), purchase a [premium subscription](/support/premium.md), [join the support server](https://discord.com/invite/hNQWVVC) (which besides support you can also read news and changelogs, participate in [volunteer programs](/support/volunteering.md) and in giveaways, meet the person who wrote this and the developer behind all this, among more), or if you started using MoonlightBot less than 7 days ago, the durations for how long you can set a temporary action to last change. Refer to the chart to see how long you can set a temporary action for:
+
+| Status              | Maximum Duration    |
+| ----------------- | ---------------- |
+| Default               |7 Days   |
+| Default [(Beta Branch)](/support/beta.md) |14 Days |
+| Joining [the support server](https://discord.com/invite/hNQWVVC) or for your first 7 days |30 Days |
+| Being a [Premium Subscriber (Basic Tier)](/support/premium.md#tiers) |90 Days  |
+| Being a [Premium Subscriber (Advanced and greater Tiers)](/support/premium.md#tiers) |365 Days  |
+| Having [Voted Recently](/support/upvote-moonlightbot.md) |Double your current maximum |
+
+{% hint style="info" %}
+When your maximum allowed duration changes, we recommend that you use [`/list-temproles`](/role-management-commands/list-temproles.md) to check if all of your temporary and pause roles are [sustained](/start-up/faqs.md#how-does-the-temprole-sustain-mechanic-work).
+{% endhint %}
 
 ## Required permissions
 
-MoonlightBot requires the following permissions to successfully execute this command:
+MoonlightBot requires the following permissions to execute this command successfully:
 
 *No specific permissions required*
 
 By default, a user is required to have the following permissions to use this command:
 
 *No specific permissions required*
 
-For more information on editing permission requirements for specific users/roles, refer to the [permissions tutorial](<linkToPermissionsTutorial>)
+For more information on editing permission requirements for specific users/roles, refer to the [permissions tutorial](/start-up/permission-tutorial.md)
 
 ## Syntax
 
 ```text
-/check-duration [debug]
+/check-duration
 ```
 
-### Options
-
-* `debug`: <TODO: describe>
-
 ## Logs
 
 *This command does not trigger any log events.*
 
-For more information on setting up those logs, refer to the [log setup tutorial](<linkToLogTutorial>)
\ No newline at end of file
+For more information on setting up those logs, refer to the [log setup tutorial](/advanced/list-of-log-names.md)
```

Comment:
The list of log names isn't the tutorial, same thing as the other autogenerated. Someone will take care of this replacement in bulk once the doc is full

Reply:
this